From 50ff383bd6fe384f43345ce127f757da3a04ed83 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:33:37 +0100
Subject: [PATCH 8/8] ia64: Add rseq_slice_yield() syscall

rseq_slice_yield() was added after ia64 was removed from the kernel.

Fixes: 99d2592023e5 ("rseq: Implement sys_rseq_slice_yield()")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 36c2d6cba49b..1d16016ebdce 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -391,3 +391,4 @@
 468	common	file_getattr			sys_file_getattr
 469	common	file_setattr			sys_file_setattr
 470	common	listns				sys_listns
+471	common	rseq_slice_yield		sys_rseq_slice_yield
-- 
2.52.0

